src/lib/homestore_backend/index_kv.cpp

```diff
@@ -75,7 +75,7 @@ HSHomeObject::get_blob_from_index_table(shared< BlobIndexTable > index_table, sh
 
     // blob get API
     if (state == BlobState::ALIVE) {
-        if (pbas == tombstone_pbas) { return folly::makeUnexpected(BlobError::INDEX_ERROR); }
+        if (pbas == tombstone_pbas) { return folly::makeUnexpected(BlobError::UNKNOWN_BLOB); }
```

Comment:
Cannot make a comment on L63,  but do we really need the `state` in the api call?
It's intent was trying to distinguish between `on purpose reading tombstone` or `trying to read a live blob but hit tombstone`.

can we do 
```
 if (pbas == tombstone_pbas) 
     return folly::makeUnexpected(BlobError::UNKNOWN_BLOB); 
else 
     return pbas;
```